### PR TITLE
Added --chain polkadot support

### LINKED ISSUE #1297: support --chain polkadot flag option
## Task summary

- similar to `--chain gssmr`, `--chain ksmcc`, we should add a `--chain polkadot` flag option for the `--chain` flag that uses the genesis file and config file in `chain/polkadot`

### PR DESCRIPTION
**Pull request by anthdm**
## Changes

- added support for `--chain polkadot`

## Tests

```

```

## Checklist

- [ ] I have read [CODE_OF_CONDUCT](https://github.com/ChainSafe/gossamer/blob/development/.github/CODE_OF_CONDUCT.md) and [CONTRIBUTING](https://github.com/ChainSafe/gossamer/blob/development/.github/CONTRIBUTING.md) 
- [ ] I have provided as much information as possible and necessary
- [ ] I have reviewed my own pull request before requesting a review
- [ ] All integration tests and required coverage checks are passing

## Issues

- closes #1297

### CODE REVIEW

**Review by noot — COMMENTED** [review]
looks good - please add polkadot genesis file!

File: chain/polkadot/config.toml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+[init]
+genesis-raw = "./chain/polkadot/genesis-raw.json"
+
+[account]
+key = ""
+unlock = ""
+
+[core]
+roles = 4
```
Comment (noot) [inline chain/polkadot/config.toml:23]: change to 1 (full node)

File: chain/polkadot/defaults.go [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+	DefaultKey = string("")
+	// DefaultUnlock Default account unlock
+	DefaultUnlock = string("")
+
+	// CoreConfig
+
+	// DefaultAuthority is true if the node is a block producer and a grandpa authority
+	DefaultAuthority = true
+	// DefaultRoles Default node roles
+	DefaultRoles = byte(4) // authority node (see Table D.2)
```
Comment (noot) [inline chain/polkadot/defaults.go:56]: same here

**Comment by anthdm** [comment]
Still having this issue:

```
EROR[02-03|18:50:25] failed to initialize node                pkg=cmd error="failed to fetch genesis babe configuration: cannot decode invalid boolean" caller=main.go:296
failed to fetch genesis babe configuration: cannot decode invalid boolean
```

I Will look into this tomorrow morning

**Review by RyRy79261 — APPROVED** [review]

File: cmd/gossamer/main_test.go [hunk 3]
```diff
@@ -212,6 +212,7 @@ func TestMain(m *testing.M) {
 	}
 	defaultGssmrConfigPath = "../../chain/gssmr/config.toml"
 	defaultKusamaConfigPath = "../../chain/ksmcc/config.toml"
+	defaultPolkadotConfigPath = "../../chain/polkadot/config.toml"
```
Comment (noot) [inline cmd/gossamer/main_test.go:215]: is this used anywhere? if you could add a test case that'd be good

**Review by noot — APPROVED** [review]

**Pull request merged**